CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * By default, `system.abort-jobs-on-terminate` is false when running `java -jar cromwell.jar server`, and true when running `java -jar cromwell.jar run <wdl> <inputs>`.
 * Enable WDL imports when running in Single Workflow Runner Mode.
 * Support for sub workflows
+* Support declarations as graph nodes
```

Comment:
Yeah I wasn't sure how to describe what it enables in a clear expect from giving an example :/